boards/arm/npcx7m6fb_evb/support/npcx_monitor.cfg

```diff
@@ -0,0 +1,70 @@
+# Script for program/verify specific embedded flash in npcx ec series 
```

Comment:
What language is this script? Can you add a comment?

Reply:
Done